src/v2/view-builder/views/ov/NumberChallengePushView.js

```diff
@@ -0,0 +1,77 @@
+import { _, loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import polling from '../shared/polling';
+import { WARNING_TIMEOUT } from '../../utils/Constants';
+import ResendNumberChallengeView from './ResendNumberChallengeView';
+import NumberChallengePhoneView from './NumberChallengePhoneView';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    noButtonBar: true,
+
+    className: 'okta-verify-number-challenge',
+
+    events: {
+      'click a.resend-number-challenge': 'handleResendNumberChallenge'
+    },
+
+    handleResendNumberChallenge () {
+      this.options.appState.trigger('invokeAction', 'currentAuthenticator-resend');
+      // remove the warning
+      this.stopPoll();
+      // restart polling
+      this.startPoll();
+    },
+
+    title () {
+      return loc('oie.okta_verify.push.sent', 'login');
+    },
+
+    initialize () {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.add(NumberChallengePhoneView);
+    },
+
+    triggerAfterError () {
+      BaseForm.prototype.triggerAfterError.apply(this, arguments);
+      this.stopPolling();
+      this.$el.find('.o-form-fieldset-container').empty();
```

Comment:
@haishengwu-okta see, for this i added a common handler in baseForm to clear fieldset content in my last PollView  PR.. do you think we should add a common handler now ? 

```
this.trigger('handleClearFieldsetContent');
```

Reply:
- curious why need to hide the form?
- would it be more generic like `trigger('hideFormContent')`

Reply:
It's not all form content.. it's just the fieldset.